src/Templating/MutableTemplateRegistry.php

```diff
@@ -18,13 +18,23 @@
  */
 final class MutableTemplateRegistry extends AbstractTemplateRegistry implements MutableTemplateRegistryInterface
 {
-    public function setTemplates(array $templates): void
+    public function setTemplates(array $templates, bool $override = true): void
```

Comment:
As this is a **registry** compontent, we should stick to common names like `add*`, `set*` or `replace*`.

IMHO `configure` is too generic. You can't guess what the method does without looking at the code.

Reply:
I agree. If the method performs a merge, we could explain this in the description.